implementation/src/main/java/io/smallrye/mutiny/helpers/EmptyUniSubscription.java

```diff
@@ -10,11 +10,19 @@
 public class EmptyUniSubscription implements UniSubscription {
 
     /**
+     * A subscription that has been cancelled.
      * The instance that can be shared.
      * Calling {@link #cancel()} is a no-op.
      */
     public static final UniSubscription CANCELLED = new EmptyUniSubscription();
 
+    /**
+     * A subscription that has been done.
+     * The instance that can be shared.
+     * Calling {@link #cancel()} is a no-op.
+     */
+    public static final UniSubscription DONE = new EmptyUniSubscription();
```

Comment:
Maybe define "DONE"

Reply:
This was added because some operators had assumptions on `CANCELLED`, so there was a need for another no-op. I'll clarify the javadoc.